sys/cddl/dev/dtrace/dtrace_ioctl.c

```diff
@@ -333,26 +387,26 @@ dtrace_ioctl(struct cdev *dev, u_long cmd, caddr_t addr,
 		/*
 		 * We have our snapshot; now copy it out.
 		 */
-		if (copyout(buf->dtb_xamot, desc.dtbd_data,
+		if (copyout_c(buf->dtb_xamot, desc_dtbd_data,
```

Comment:
We are moving towards `EXPLICIT_USER_ACCESS` everywhere. It aliases normal copyin/out to copyin/out_c so you can remove the _c and `#define EXPLICIT_USER_ACCESS` before includes. This means that copyin/out will need to use capabilities by default but you are already doing this here.